docs/migrations/README.md

```diff
@@ -0,0 +1,339 @@
+# Update to Cosmos-SDK v0.40
```

Comment:
The README shouldn't be that long, it should only include a table of contents.

Based on what I'm reading here, I think the ideal way would be:
1. one page for upgrading modules
2. one page for upgrading the app
3. one page for a live chain upgrade